lib/geo_postgis.ex

```diff
@@ -432,4 +432,11 @@ defmodule Geo.PostGIS do
   defmacro st_make_point(x, y, z, m) do
     quote do: fragment("ST_MakePoint(?, ?, ?, ?)", unquote(x), unquote(y), unquote(z), unquote(m))
   end
+
+  defmacro st_make_envelope(xMin, yMin, xMax, yMax) do
+    quote do: fragment("ST_MakeEnvelope(?, ?, ?, ?)", unquote(xMin), unquote(yMin), unquote(xMax), unquote(yMax))
+  end
```

Comment:
Fix to comply with `mix format` style:

```suggestion
  end

```